lib/open_food_network/feature_toggle.rb

```diff
@@ -38,10 +49,23 @@ module FeatureToggle
       DESC
     }.freeze
 
+    # Features you would like to be enabled to start with.
+    #
+    # Copy features here that were activated in a migration so that new
+    # instances, development and test environments have the feature active.
+    ACTIVE_BY_DEFAULT = {
+      "background_reports" => <<~DESC,
+        Generate reports in a background process to limit memory consumption.
```

Comment:
We don't need the description here but a hash lookup by key is more convenient than searching in an array and including the description enables simple copy and paste from the feature list above.